bom/pom.xml

```diff
@@ -835,6 +835,16 @@
                 <artifactId>helidon-messaging-kafka</artifactId>
                 <version>${helidon.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.helidon.messaging.jms</groupId>
+                <artifactId>helidon-messaging-jms</artifactId>
+                <version>${helidon.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.helidon.messaging.aq</groupId>
+                <artifactId>helidon-messaging-aq</artifactId>
+                <version>${helidon.version}</version>
+            </dependency>
```

Comment:
Hello, I couldn't see this being resolved. May I know if its available for public?

Reply:
Hi, helidon-messaging-aq is part of this PR, it needs to be accepted, merged and then released with new version of Helidon

Reply:
Got it, thanks for the update.